code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -427,6 +427,21 @@
 	startswith = list(/obj/item/reagent_containers/pill/paracetamol = 21)
 	wrapper_color = "#a2819e"
 
+/obj/item/storage/pill_bottle/stabiliser
+	name = "pill bottle (prescribed stabiliser)"
+	desc = "A pill bottle given primarially to the injured whilst doctors can work on more urgent matters. Take with caution, unless you have a death wish."
+
+	startswith = list(
+			/obj/item/reagent_containers/pill/inaprovaline = 6,
+			/obj/item/reagent_containers/pill/oxycodone = 6,
+			/obj/item/reagent_containers/pill/dexalin_plus = 2,
```

Comment:
now this is just medical powergamer's dream

Reply:
**it's a draft!!!**